addons/web_editor/static/src/scss/web_editor.common.scss

```diff
@@ -669,6 +669,35 @@ blockquote {
                     color: $dropdown-link-disabled-color !important;
                 }
             }
+            .btn-link {
+                // Some search autocomplete results use btn-link for extra
+                // information that can be navigated to. (e.g. products give
+                // access to their categories)
+                // These need to be reset in case a dark palette is applied.
+                color: $link-color;
+
+                &:hover {
+                    color: $link-hover-color;
+                }
```

Comment:
Now that we are with BS5, I think there actually is the btn-link variables I saw before: 
![image](https://user-images.githubusercontent.com/10338094/182363407-fef614ae-4f71-4b6f-9720-89e3784e28c2.png)

Maybe check the other variables used in that file too at the same time :+1:

Reply:
I did not spot any other variable that needs replacement within `web_editor.common.scss`.
(The `a` within `text-muted` gets intentionally reset to `$link*-color` and `$link-hover-color`)

Now using the `$btn-link*-color`s for resetting `.btn-link` colors.

Reply:
@qsm-odoo Applied changes and re-tested on runbot.